Cargo.toml

```diff
@@ -34,6 +34,8 @@ indoc = "0.3.3"
 [build-dependencies]
 regex = "1.1.6"
 version_check = "0.1.5"
+serde = { version = "1.0.91", features = ["derive"] }
+serde_json = "1.0.39"
```

Comment:
Do we really need the specific version of serde?

Reply:
Currently no, but it's only a build dependency and it avoids problems when using features from newer versions without checking `-Z minimal-versions` for compatibility